contracts/cryptoblades.sol

```diff
@@ -486,7 +495,7 @@ contract CryptoBlades is Initializable, AccessControlUpgradeable {
         (,, uint256 fromUserWallet) =
             getSkillToSubtract(
                 0,
-                tokenRewards[msg.sender],
+                0,
```

Comment:
I don't fully understand this change, also, to avoid magic 0, it would be nice to add a comment explaining why it's a zero

Reply:
getSkillToSubtract takes 3 arguments `uint256 _inGameOnlyFunds, uint256 _tokenRewards, uint256 _skillNeeded`, I set _tokenRewards to 0 so everything needs to come from the wallet.